docs/topic-guides/blendable-web-apps/uniform-design-system.md

```diff
@@ -2,17 +2,15 @@
 
 ## Introduction
 
-Uniform design system is a user interface toolkit created by Starcounter. It allows authoring of blendable apps that have a common look and feel. 
-
-Uniform is applied by the Starcounter host. It controls the look and feel of native HTML5, typical UI patterns, and components. It is a link between app authors and solution owners.
+Uniform design system is a user interface toolkit created by Starcounter. It gives a common look and feel to blendable apps. It is a link between app authors and solution owners.
 
 Unlike many design systems, Uniform is open-ended and themeable. It can be configured, extended or replaced in parts to achieve anything that is possible with HTML, CSS, and JavaScript.
 
-![Decision tree of theming Uniform design system](../../.gitbook/assets/uniform-decision-tree.png)
-
 ## Mental model of client-side rendering by the Starcounter host
 
-Starcounter host follows a layered design that allows controlling the look and feel of all apps. These layers are:
+Starcounter host follows a layered design that separates the responsibilities of the app author, the design system and the solution owner. Uniform fits in this design as the middle layer. 
+
+The full design consists of:
```

Comment:
I don't get what you try to express here. Could you try to elaborate more, or call me on HO/slack?